language/libra-vm/src/libra_vm.rs

```diff
@@ -632,12 +632,13 @@ impl VMVerifier for LibraVM {
         &self,
         transaction: SignedTransaction,
         state_view: &dyn StateView,
-    ) -> Option<VMStatus> {
+    ) -> VMValidatorResult {
         let data_cache = BlockDataCache::new(state_view);
+        let gas_price = transaction.gas_unit_price();
```

Comment:
- TODO for normalization?
- Will this eventually be returned from the prologue + read from the return value of `verify_transaction_impl` below?